rio/srcrobot/commands/TurnInPlace.py

```diff
@@ -30,6 +45,16 @@ def getRotationValue(self):
             return rotationStick*Constants.Swerve.maxAngularVelocity
         else:
             self.angularvelMRadiansPerSecond = -self.turnPID.calculate(self.s_Swerve.getHeading().radians(), self.angle)
+            self.angularvelMRadiansPerSecond = math.copysign(math.fabs(self.angularvelMRadiansPerSecond) + self.kBarelyNotTurnFeedforward, self.angularvelMRadiansPerSecond)
```

Comment:
When eventually the PID calculated value becomes ~0, since we are adding the feedforward, wouldn't the robot never stop (0+0.2)? Or is this going off of that the command would get unscheduled once the PID is on target?

Reply:
The desired outcome is that being on target with stop it. However, see the last comment I added to the code with regard to modifying the feed forward close to target.